lib/convict.js

```diff
@@ -501,27 +511,53 @@ let convict = function convict(def) {
      */
     validate: function(options) {
       options = options || {};
-      options.strict = options.strict || false;
-      let errors = validate(this._instance, this._schema, [],options.strict);
 
-      if (errors.length) {
-        let errBuf = '';
+      if (ALLOWED_OPTION_STRICT in options) {
```

Comment:
This is an error here: `ALLOWED_OPTION_STRICT` cannot be in `options`. `ALLOWED_OPTION_STRICT` can only be in `options.allowed`.

You have done one search-and-replace too many! Here it should be: `if ('strict' in options)`.